### PR TITLE
Change default login and signin method to POST #542

### PR DESCRIPTION
**Pull request by jjunineuro**
Improves the object to perform the Login using the POST method as a request.

### CODE REVIEW

**Comment by nstrelow** [comment]
I get an error when building android, after including this new commit

```
../../../AppData/Local/Pub/Cache/git/Parse-SDK-Flutter-4f6474058ec7eee7bd0426a9596f0092fb3b1c3f/packages/dart/lib/src/objects/parse_user.dart:223:9: Error: The argument type 'Map<String, dynamic>' can't be assigned to the parameter type 'String'.
 - 'Map' is from 'dart:core'.
		data: queryParams,
		      ^


FAILURE: Build failed with an exception.
```

File: packages/dart/lib/src/objects/parse_user.dart [hunk 1]
```diff
@@ -216,11 +216,11 @@ class ParseUser extends ParseObject implements ParseCloneable {
...
       };
       final String installationId = await _getInstallationId();
-      final Uri url = getSanitisedUri(_client, '$keyEndPointLogin',
-          queryParams: queryParams);
+      final Uri url = getSanitisedUri(_client, '$keyEndPointLogin');
       _saveChanges();
-      final ParseNetworkResponse response = await _client.get(
+      final ParseNetworkResponse response = await _client.post(
         url.toString(),
+		data: queryParams,
```
Comment (nstrelow) [inline packages/dart/lib/src/objects/parse_user.dart:223]: You could try keeping the line indentation here

**Comment by jjunineuro** [comment]
Hi @nstrelow ,

I did the repair on the indentation as well as converting the variable "queryParams" to a JSON. By default, Dio receives a String in "data" and "queryParams" is a Map.
So avoid the mistake you mentioned.

[https://github.com/parse-community/Parse-SDK-Flutter/pull/545](https://github.com/parse-community/Parse-SDK-Flutter/pull/545)

**Comment by RodrigoSMarques** [comment]
@jjunineuro 
I will reverse the commit.

**Comment by nstrelow** [comment]
Thanks :D

Also for #545, I think it should point to the `development` branch not the `master` branch.

@jjunineuro @RodrigoSMarques Thanks for the quick action :D

**Pull request merged**